test-types/basic-matchers.test.ts

```diff
@@ -0,0 +1,428 @@
+/**
+ * @type ./empty.d.ts
+ */
+
+import {expectError, expectType} from 'mlh-tsd';
+import {expect, jest} from '@jest/globals';
+import type {Mock} from 'jest-mock';
+
+describe('', () => {
+  it('', () => {
+    /* Corrections of previous typings */
+
+    // FIXME: TSD's limitations. Cannot run a function if it does not exist?
+    // expectError<jest.Matchers<string>>(expect('').not.not);
```

Comment:
```suggestion
    expectType<void>(expect('').not.not);
```

maybe?

Reply:
Actually, it does not work. Tests fail. It actually returns `Matchers<string>` and not `undefined`.

Reply:
that seems like a bug in the types. gonna be great to have tests for this so we get them correct! 😀

Reply:
So, should I change it to `expectType<undefined>(expect('').not.not);` so that it fails?

Reply:
Leave it for now, then we can go over and fix it later. Landing these tests as is without starting to change other code seems like a great first step to me